Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -8388,4 +8388,31 @@ public void TODO_Actions_ReResolvingBindings_DoesNotAllocate_IfXXX()
     {
         Assert.Fail();
     }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_Buttons_CanTransitionToDifferentActuationControl_WhilePerformed()
```

Comment:
Maybe `Actions_OnActionWithMultipleBindings_CanTransitionFromOneActuatedControlToAnother` (there's a couple OnActionWithMultipleBindings tests already)

Reply:
Fixed, moved the test closer to them